components/tenant-mgt/org.wso2.carbon.tenant.keystore.mgt/src/main/java/org/wso2/carbon/keystore/mgt/KeyStoreGenerator.java

```diff
@@ -225,7 +225,7 @@ private void persistKeyStore(KeyStore keyStore, X509Certificate PKCertificate)
 
             String keyStoreName = generateKSNameFromDomainName();
             // Use the keystore using the keystore admin
-            KeyStoreAdmin keystoreAdmin = new KeyStoreAdmin(tenantId, govRegistry);
```

Comment:
We can remove registry object from this class

Reply:
Addressed with: https://github.com/wso2/carbon-multitenancy/compare/2ba63f8e7fd72fd3ab97db45aff68399b19eafa3..b6ac424041c370652d72e1e0fadb707ea615ee20